releng/prepare-relnotes

```diff
@@ -21,7 +21,7 @@
 export prev=$1
 export next=$2
 
-export contributors=`git shortlog v${prev}..develop -sn --no-merges | awk -F'\t' '{print $2}' | tr -s "\\n" "*" | sed -e 's@*@, @g' -e 's@, $@\n@g'`
+export contributors=`git shortlog v${prev}..develop -sn --no-merges | cut -d'\t' -f 2 | tr -s "\\n" "*" | sed -e 's@*@, @g' -e 's@, $@\n@g'`
```

Comment:
That's ok for me, as cut call is lighter than awk :)

Reply:
This might be the only change that is worth having compared to the others, although the impact will not be noticeable because it's so small and this script is used a few times by the developers.